src/mechanics/interpolation/CellInterpolationBase.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include<eigen3/Eigen/Dense>
+#include "TypeDefs.h"
+
+namespace NuTo
+{
+
+class CellInterpolationBase
+{
+public:
+    CellInterpolationBase(){}
+
+    //! @brief extracts all node values of this element
+    //! @remark virtual to make it testable
```

Comment:
What is meant by "virtual to make it testable"? Isn't it "virtual to make it polymorphic"? (which would be a superfluous remark).